src/Appwrite/Platform/Modules/Databases/Http/Databases/Create.php

```
@@ -108,33 +108,33 @@ private function constructDatabaseDSNFromProjectDatabase(string $databasetype, $
         if ($index !== false) {
             $selectedDsn = $databases[$index];
         } else {
-            if (!empty($dsn)) {
+            if (! empty($dsn) && ! empty($databaseSharedTables)) {
                 $beforeFilter = \array_values($databases);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$beforeFilter'. (undefined)

(UnusedLocalVariable)